integreat_cms/textlab_api/textlab_api_client.py

```diff
@@ -15,6 +15,21 @@
 logger = logging.getLogger(__name__)
 
 
+"""
+Mapping between fields in Textlab API response containing feedback data
+and the corresponding category names used in the CMS
+"""
```

Comment:
Nitpick: this should probably be a normal comment, as docstrings are only used for module, class, method or function documentation (https://peps.python.org/pep-0257/#what-is-a-docstring)